### PR TITLE
Initial Travis CI support

### PR DESCRIPTION
**Pull request by ridv**
* Added initial support for minimal Travis CI.
* Deleted Aurora deb file as it's out of date.
* Moved from glide to dep due to the fact that gorealis uses dep. Therefore, to be compatible with goorealis' dependency management, we have to move to dep. This leaves us in a good place to migrate both to mods once they stabilize.

### CODE REVIEW

**Comment by sjlee** [comment]
This is great. Thanks for the work! Could you also consider adding a CI status indicator in the README? That would complement this nicely.

**Comment by ridv** [comment]
@sjlee added a badge to the PR. Thanks for pointing that out!

**Comment by kkrishna** [comment]
Thanks @rdelval  for the PR!
There are internal dependent repos for plugins. So transition from glide to dep would require for those as well. We maintain status quo until we switch to mods or switch to dep for all repos. 
I see no hard dependency in keeping compatibility with gorealis dependency management.

**Comment by ridv** [comment]
The hard dependency here was using the same build system to allow it to create the vendor folder correctly. For now I've put a workaround to maintain compatibility but I still think moving to dep is the safer move. Hopefully mods become stable within the next 6 months and we can migrate to them.

The initial build was successful so we're ready to go https://travis-ci.org/paypal/dce-go/builds/448598566

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+language: go
+
+go:
+  - "1.10.x"
+
+install:
+   - glide install
+   # Needed in order for compilation to work since gorealis uses dep and we use glide.
+   - rm -r vendor/github.com/paypal/gorealis/vendor/git.apache.org/thrift.git
```
Comment (kkrishna) [inline .travis.yml]: use strip-vendor to get rid of nested vendor directories from dependencies as opposed to  manual delete :
```
glide install --strip-vendor
```

Comment (ridv) [reply .travis.yml]: https://travis-ci.org/paypal/dce-go/builds/448915117?utm_source=github_status&utm_medium=notification

**Comment by ridv** [comment]
@kkrishna I had to rollback to version v0.13.1 as per https://github.com/Masterminds/glide/issues/1025 in order for this to work. It's ready to go. @JiaminZhu if you have time, look this over and squash and merge when ready 🎉

**Pull request merged**